buildchow/templates/defaults.yaml

```diff
@@ -0,0 +1,37 @@
+- defaults:
+    name: global
+    disabled: false
```

Comment:
Ah, then however you want to encode it, do this: 
if (master) then enabled by default; else if (_4.3.mars / _9.0.mars) then disabled by default;